lib/ansible/executor/task_executor.py

```diff
@@ -738,7 +738,7 @@ def _poll_async_result(self, result, templar, task_vars=None):
         # we need the 'normal' action handler for the status check, so get it
         # now via the action_loader
         async_handler = self._shared_loader_obj.action_loader.get(
-            'async_status',
+            'ansible.legacy.async_status',
```

Comment:
Just happened to notice this while looking through the code. This matches what we are doing in other action plugins. Probably not really important here.